BookingProject/BookingWebAPI/BookingWebApi.Application/Services/ApartamentService.cs

```diff
@@ -27,11 +29,16 @@ public async Task<Result<ApartamentPostDto>> CreateApartament(ApartamentPostDto
                 return Result<ApartamentPostDto>.Failure("User id is required");
             }
 
-            if(!await _appUserRepository.IsUserExist(userId))
+            if(!await _appUserRepository.UserExists(userId))
             {
                 return Result<ApartamentPostDto>.Failure("User wasn`t found");
             }
 
+            if(!await _userManager.IsInRoleAsync(userId, "Host"))
```

Comment:
Create on domain layer `RoleConstants`. And there you can make a constant for your role. Never use simple strings because it's really baaaaad - you can do typos. Or while refactoring you will have to replace it in the whole project. This causes problems in the future

Reply:
fixed